components/edit_post_modal/edit_post_modal.jsx

```diff
@@ -491,10 +496,15 @@ class EditPostModal extends React.PureComponent {
                         disabled={this.isSaveDisabled()}
                         onClick={this.handleEdit}
                     >
-                        <FormattedMessage
-                            id='edit_post.save'
-                            defaultMessage='Save'
-                        />
+                        <span
+                            id='editModalFooter'
```

Comment:
Could you remove this ID since we're not using it anywhere else? We're trying to cut down on unused IDs since people sometimes use them for CSS styling when they shouldn't.